cmd/clusterawsadm/cmd/alpha/ec2/ec2.go

```diff
@@ -40,30 +39,38 @@ func RootCmd() *cobra.Command {
 
 func consoleOutputCmd() *cobra.Command {
 	newCmd := &cobra.Command{
-		Use:   "console-output",
+		Use:   "console-output [EC2 Machine ID]",
 		Short: "Get console output for host",
-		Long:  "Get console output for host",
-		Args:  cobra.ExactArgs(1),
-		Run: func(cmd *cobra.Command, args []string) {
+		Long: `Get console output for host.
+You must enter a Machine ID for machine you want to get console output. Machine ID can be obtained by following https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
```

Comment:
This link may not be a correct one. Is there a way to obtain Machine ID without accessing the machine?

Reply:
I think we should kill this atm. It doesn't work very well (my fault), and that command itself only works for Generation 5 instances as it later turned out.